ARCHIVE/ICCDEF/itkMultiResolutionICCDeformableRegistration.h

```diff
@@ -294,7 +294,7 @@ class MultiResolutionICCDeformableRegistration :
    *
    * \sa ProcessObject::VerifyInputInformation
    */
-  void VerifyInputInformation() override
+  void VerifyInputInformation() const override
```

Comment:
@hjmjohnson ITKv5 has added a new backwards comaptible flag to allow building with ITKv4 by using ITKV5_CONST macro that is defined as nothing in ITKv4.